man/mblaze.7

```diff
@@ -3,81 +3,81 @@
 .Os
 .Sh NAME
 .Nm mblaze
-.Nd introduction to mblaze
+.Nd introduction to the mblaze message system
 .Sh DESCRIPTION
 The
 .Nm
-message system is a set of Unix utilities to deal with
-mail kept in Maildir folders.
+message system is a set of Unix utilities for processing and
+interacting with mail messages which are stored in maildir folders.
 .Pp
 Its design is roughly inspired by MH, the RAND Message Handling
 System, but it is a complete implementation from scratch.
 .Pp
 .Nm
-consists of these Unix tools that each do one job:
+consists of these Unix utilities that each do one job:
+.Pp
 .Bl -tag -width 11n -compact
 .It Xr maddr 1
-extract addresses from mail
+extract mail addresses from messages
 .It Xr magrep 1
-find mails matching a pattern
+print messages matching a pattern
 .It Xr mbnc 1
-bounces mail
+bounce messages
 .It Xr mcom 1
-compose and send mail
+compose and send messages
 .It Xr mdeliver 1
-deliver messages or import mailboxes
+deliver messages or import mbox file
 .It Xr mdirs 1
-find Maildir folders
+list maildir folders, recursively
 .It Xr mexport 1
-export Maildir folders as mailboxes
+export messages as mbox file
 .It Xr mflag 1
-change flags (marks) of mail
+manipulate maildir flags
 .It Xr mflow 1
-reflow format=flowed plain text mails
+reflow format=flowed plain text messages
 .It Xr mfwd 1
-forward mail
+forward messages
 .It Xr mgenmid 1
-generate Message-IDs
+generate a Message-ID
 .It Xr mhdr 1
-extract mail headers
+print message headers
 .It Xr minc 1
-incorporate new mail
+incorporate new messages
 .It Xr mless 1
-conveniently read mail in
+conveniently read messages in
 .Xr less 1
 .It Xr mlist 1
-list and filter mail messages
+list and filter messages
 .It Xr mmime 1
-create MIME messages
+encode MIME messages
```

Comment:
I prefer `create` here, or perhaps `encode messages as MIME`?

Reply:
I'm happy with either, so long as we change SYNPOSIS in `mmime.1` in line with it. (`mmime.1` currently says 'encode MIME mails'.)

Reply:
RFC 2047 refers to 'MIME messages', so I'm good with 'create MIME messages' if you are.

Reply:
Then `create MIME messages` in both.